pkg/observer/data_stats.go

```diff
@@ -0,0 +1,65 @@
+package observer
```

Comment:
SPDX header is missing here (a CI check for that would be nice, but that's unrelated to this PR).

Reply:
will fix, thanks